src/js/components/Footer.jsx

```diff
@@ -1,6 +1,36 @@
 const Footer = () => {
   return (
     <footer className="component Footer">
+      <style>
```

Comment:
Would be nice if we can maintain all the styling in the Footer.scss

Reply:
I agree we should move those styles to the scss (or we can use inline styles sparingly